app/models/organisation.rb

```diff
@@ -1,6 +1,4 @@
 class Organisation < ActiveRecord::Base
-  belongs_to :parent, class_name: Organisation, foreign_key: 'parent_id'
```

Comment:
Ah, ok, I reviewed the overall PR rather than individual commits and missed that!